src/layouts/Root/components/StyledNavigationLink.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { Link } from 'gatsby';
+import { getColor } from '@zendeskgarden/react-theming';
+
+export const StyledNavigationLink = styled(Link).attrs({ activeClassName: 'is-active' })`
+  border-radius: ${p => p.theme.borderRadii.md};
+  padding: ${p => p.theme.space.base * 1.5}px ${p => p.theme.space.xs};
+  color: ${p => p.theme.colors.foreground};
+
+  &:focus,
+  &:hover,
+  &:active {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &.is-active {
```

Comment:
Perhaps `is-current` to differentiate from `:active`? Avoids future confusion. Also aligns with https://github.com/zendeskgarden/react-components/blob/master/packages/chrome/src/styled/nav/StyledNavItem.ts.

Reply:
Updated.